app/Http/Middleware/LogRequests.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Auth\AuthenticationException;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Log as SystemLog;
+
+class LogRequests
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param \Closure(Request): (Response) $next
+     * @throws AuthenticationException
+     */
+    public function handle(Request $request, Closure $next): Response
+    {
+        if (Auth::check() && $request->isMethod('post')) {
+            SystemLog::debug('User #' . user()->id . ' sent request: path='
```

Comment:
Does this only log in debug mode or also in production?

Reply:
Same as before, the files just have been renamed. I think debug statements won't have an effect in production.

Reply:
I checked it, a DEBUG log is created and this middleware is active in production. But it is set in multiple routes to not log (like when someone votes on a general assembly, authentication etc.).